### DISCUSSION TITLE
avif format support

### DISCUSSION THREAD

**Discussion by mansai** [AvaloniaUI/Avalonia#14643] [Q&A]
How to display avif format image
test file
https://files.catbox.moe/w3meta.avif
https://towcardimages.s3.amazonaws.com/001/3/00003-hmsvictory-animated-test2.avif

**Comment by thevortexcloud** [comment]
It sounds like SkiaSharp should support Avif. Does it not work now?

**Comment by aizigao** [comment]
need this too

**Reply by stevemonaco** [reply]
Avalonia doesn't ship its own image decoders. It relies on libraries like Skia(Sharp) for that. AVIF decoding seems to have entered Skia at milestone 106: https://skia.googlesource.com/skia/+/refs/heads/main/RELEASE_NOTES.md. SkiaSharp 2 (which Avalonia 11 uses by default) is on milestone 88. You should try updating to SkiaSharp 3 which is on milestone 119.
